fearless/Modules/ChainAccountBalanceList/ChainAccountBalanceListViewController.swift

```diff
@@ -0,0 +1,121 @@
+import UIKit
+import SoraFoundation
+
+final class ChainAccountBalanceListViewController: UIViewController, ViewHolder {
+    typealias RootViewType = ChainAccountBalanceListViewLayout
```

Comment:
what is `Type` here for? no needed definitely

Reply:
@posplaw please explain. this class conforms to protocol ViewHolder that have associatedtype